architecture-building-systems/CityEnergyAnalyst
# 1813 Zone Helper

This PR solves issue #1813 which extracts the Zone.shp file from OSM.

It also creates the occupancy and the age file.

This is part of some work we need to do to create the "new project" feature inside the new CEA dashboard.

How to test:
1. uise the site.shp file stored in building_geometry
2. run zone_helper.py
3. check that the outputs: zone.shp, occupancy.dbf and age.dbf are created

@reyery please check that it runs in your computer.
